src/Administration/Resources/app/administration/src/app/filter.factory.js

```diff
@@ -0,0 +1,59 @@
+import _ from 'lodash';
+
```

Comment:
I would activate EsLint. Because in this file are many errors which needs to be fixed.